x-pack/plugins/security_solution/public/common/components/exceptions/viewer/exception_item_card/index.tsx

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  EuiPanel,
+  EuiFlexGroup,
+  EuiCommentProps,
+  EuiCommentList,
+  EuiAccordion,
+  EuiFlexItem,
+  EuiText,
+  useEuiTheme,
+} from '@elastic/eui';
+import React, { useMemo, useCallback } from 'react';
+import type { ExceptionListItemSchema } from '@kbn/securitysolution-io-ts-list-types';
+
+import { getFormattedComments } from '../../helpers';
+import type { ExceptionListItemIdentifiers } from '../../types';
+import * as i18n from './translations';
+import { ExceptionItemCardHeader } from './exception_item_card_header';
+import { ExceptionItemCardConditions } from './exception_item_card_conditions';
+import { ExceptionItemCardMetaInfo } from './exception_item_card_meta';
+
+export interface ExceptionItemProps {
+  loadingItemIds: ExceptionListItemIdentifiers[];
+  exceptionItem: ExceptionListItemSchema;
+  onDeleteException: (arg: ExceptionListItemIdentifiers) => void;
+  onEditException: (item: ExceptionListItemSchema) => void;
+  disableActions: boolean;
+  dataTestSubj: string;
+}
+
+const ExceptionItemCardComponent = ({
+  disableActions,
+  loadingItemIds,
+  exceptionItem,
+  onDeleteException,
+  onEditException,
+  dataTestSubj,
+}: ExceptionItemProps): JSX.Element => {
+  const { euiTheme } = useEuiTheme();
+
+  const handleDelete = useCallback((): void => {
+    onDeleteException({
+      id: exceptionItem.id,
+      namespaceType: exceptionItem.namespace_type,
+    });
+  }, [onDeleteException, exceptionItem.id, exceptionItem.namespace_type]);
+
+  const handleEdit = useCallback((): void => {
+    onEditException(exceptionItem);
+  }, [onEditException, exceptionItem]);
+
+  const formattedComments = useMemo((): EuiCommentProps[] => {
+    return getFormattedComments(exceptionItem.comments);
+  }, [exceptionItem.comments]);
+
+  const disableItemActions = useMemo((): boolean => {
+    const foundItems = loadingItemIds.filter(({ id }) => id === exceptionItem.id);
```

Comment:
nit: Because later you check for `foundItems.length > 0` maybe better here to use `.some` which will return you boolean in the first place

Reply:
Updating!